applications/nrf_desktop/description.rst

```diff
@@ -952,6 +952,24 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for information abou
 
 .. nrf_desktop_fastpair_important_end
 
+nRF54L MCUboot provisioning
+===========================
+
+nRF54L-based nRF Desktop devices enable hardware cryptography for the MCUboot bootloader.
+The public key that MCUboot uses to validate the application image is securely stored in the hardware Key Management Unit (KMU).
+In this use case, the application image is automatically signed by the |NCS| build system.
+However, the public key is not automatically provisioned to the device when programming the bootloader and the application images using the ``west flash`` command.
+
+To provision the MCUboot keys, use the ``west ncs-provision`` command before programming the bootloader and application images.
+Make sure that the provisioned public key is generated from the private key that was used to sign the application image.
+The private keys are stored in the application configuration directory of the board.
+Path to the private key is defined by the ``SB_CONFIG_BOOT_SIGNATURE_KEY_FILE`` sysbuild Kconfig option.
+You only need to provision one public key to an nRF Desktop device.
+For details, see :ref:`provisioning KMU for nRF54L devices <ug_nrf54l_developing_provision_kmu>`.
```

Comment:
Didn't we agree to  place here commends on how to provision dk?

Reply:
We discussed this idea, but eventually we desisted from doing it. More provisioning methods might be introduced in the future (I guess some of them would be done through GUI). Alternatively the provisioning might be done automatically together with programming the board. I would prefer not to maintain this doc part in scope of nRF Desktop.

The command is described on the linked `provisioning KMU for nRF54L devices` page. I could also add a single key command variant there to make it more straightforward for nRF Desktop users.